AllReadyApp/Web-App/AllReady/Features/Tasks/TaskSignupResult.cs

```diff
@@ -4,6 +4,12 @@ namespace AllReady.Features.Tasks
 {
     public class TaskSignupResult
     {
+        // TODO - Move to enum
```

Comment:
....actually, rather than thinking about enums, we're going to have to start to think about resource files.